### ISSUE TITLE
Is there code for viewpoint estimation?

### ISSUE THREAD

**Issue by yw411** [fmassa/object-detection.torch#24]
I read the paper 'CRAFTING A MULTI-TASK CNN FOR VIEWPOINT ESTIMATION' and find this pository which is used for object detection,and I saw the result of examples. So I want to know whether the text on the top left corner is the result for viewpoint estimation,including class and azimuthal angle?

**Comment by fmassa** [comment]
Yes, there is a preliminary release of the code that can be found in http://imagine.enpc.fr/~suzano-f/bmvc2016-pose/ and is based on the code in this repo.

**Comment by ardiya** [comment]
Hi @fmassa, I'm also here for CRAFTING A MULTI-TASK CNN FOR VIEWPOINT ESTIMATION. 

I tried to load Pascal3D dataset using your preliminary release, however the format seems to be different; It reads .xml annotation rather than .mat annotation on P3D dataset.

Could you share us the DataSetPascal.lua used in your paper?

**Comment by fmassa** [comment]
Hi @ardiya,
Sorry for the (long) delay in replying.

I have used the code I posted in the zip file for training the models. If the original annotations for Pascal3D are in `.mat` format, I probably converted them to `.xml` beforehand.
I'll try to have access to the computer where I have stored that data and see if the information is still there, and to see if I still have the conversion script somewhere.

**Comment by fmassa** [comment]
@ardiya I found the Matlab code I used to convert the `.mat` files to `.xml`. It requires the toolbox from Pascal VOC to be on your path.
The code is not very clean but should be easy to use.
You can find it in here https://gist.github.com/fmassa/f9f39bb07cca98d5a1b57615b4c0ea60
